tests/integration/callbacks/test_pattern_matching.py

```diff
@@ -370,3 +479,14 @@ def test_rpmc004_pattern_matching_todo(dashr):
     dashr.find_element("#add").click()
     dashr.find_element('#\\{\\"index\\"\\:1\\,\\"type\\"\\:\\"done\\"\\}').click()
     assert dashr.wait_for_text_to_equal("#totals", "1 of 1 items completed - 100%")
+
+
+def test_rpmc005_pattern_matching_graphs(dashr):
+    dashr.start_server(graphs_app)
+    dashr.select_dcc_dropdown("#country", "Cameroon")
```

Comment:
I often have a statement like ` dashr.wait_for_text_to_equal` initially before selecting anything, and then I have an `htmlDiv` with some content that renders which I can watch -- just in case there's a slight delay, in my (very limited) experience this seems to help avoid flaky tests, and can also take the place of a `sleep` statement.

Reply:
That sounds good, I'm going to make a small update to it with those elements, then I'll merge it once the final tests pass.